### PR TITLE
Rename Source filter; add data collector options

### PR DESCRIPTION
**Pull request by kruai**
This addresses [ESSNTL-2449](https://issues.redhat.com/browse/ESSNTL-2449), using the information decided upon by [ESSNTL-2443](https://issues.redhat.com/browse/ESSNTL-2443).

This renames "Source" to "Data Collector", and adds more options to the list. Here is the association of data collectors to reporters:

- insights-client -> puptoo
- subscription-manager -> rhsm-conduit
- Satellite/Discovery -> yupana

### CODE REVIEW

File: src/Utilities/constants.js [hunk 1]
```diff
@@ -11,7 +11,12 @@ export const staleness = [
     { label: 'Stale', value: 'stale' },
     { label: 'Stale warning', value: 'stale_warning' }
 ];
-export const registered = [{ label: 'Insights', value: 'insights' }, { label: 'Insights not connected', value: 'nil' }];
+export const registered = [
+    { label: 'insights-client', value: 'puptoo' },
+    { label: 'subscription-manager', value: 'rhsm-conduit' },
+    { label: 'Satellite/Discovery', value: 'yupana' },
```
Comment (roliveri) [inline src/Utilities/constants.js:17]: Is "Discovery" a data collection agent that reports through yupana (in addition to satellite), or is that one thing?

Comment (kruai) [reply src/Utilities/constants.js:17]: Yes, they both go through yupana (and I think they're separate services). The values were decided on in [ESSNTL-2443](https://issues.redhat.com/browse/ESSNTL-2443)

**Review by roliveri — APPROVED** [review]
Looks good to me.
The question I asked is for my information only.

**Review by thearifismail — APPROVED** [review]
Mapping to reporter looks good.  I did more of a testing than review due to my limited knowledge about front end.  So I deployed the front end locally to connect to the state cluster and checked a host `registered_with` `puptoo` and `rhsm-conduit` and then same host using REST API/Postman.  The second host I tested registered with Puptoo only.  Found both working.

**Comment by thearifismail** [comment]
@johnsonm325 Can you please take a look at it? 
Thanks.

**Review by karelhala — APPROVED** [review]

**Comment by karelhala** [comment]
Looking good! @kruai do you want this to be available on prod as well?

**Comment by kruai** [comment]
@karelhala It will have to wait until the API changes go in, but hopefully within the next week or so.

**Pull request merged**